docs/advanced/addons/seeder.md

```diff
@@ -1,12 +1,13 @@
 ---
-sidebar_position: 7
+sidebar_position: 4
 sidebar_label: Seeder
 title: "Seeder"
 ---
 
 <head>
   <link rel="canonical" href="https://docs.harvesterhci.io/dev/advanced/seeder"/>
 </head>
+
 _Available as of v1.2.0_
 
 The `harvester-seeder` addon lets you perform out-of-band operations on underlying nodes. 
```

Comment:
```suggestion
The `harvester-seeder` add-on lets you perform out-of-band operations on underlying nodes. 
```

Reply:
we will stick with `addon` for now since UI is already showing `Addon`